lib/metrician/middleware/request_timing.rb

```diff
@@ -143,7 +143,7 @@ def self.extract_route(controller:, path:)
       end
 
       def self.get_response_size(headers:, body:)
-        return headers[HEADER_CONTENT_LENGTH] if headers[HEADER_CONTENT_LENGTH]
+        return headers[HEADER_CONTENT_LENGTH] if headers && headers[HEADER_CONTENT_LENGTH]
```

Comment:
I think that's the only time it happens. An exception on http://github.com/Instrumental/metrician/blob/2d7c8b68b0ede4412d873ae15e0272088ed071ee/lib/metrician/middleware/request_timing.rb#L33 will cause the variables to not be set.